docs/book/intro.md

```diff
@@ -1,3 +1,4 @@
+<!-- markdownlint-disable no-inline-html first-line-heading -->
```

Comment:
Please use a redirect for this.

Reference: https://github.com/laminas/laminas-view/blob/caa49ddda37edb9977efc23cf719a75e6ce01669/mkdocs.yml#L52-L55